src/Uno.UWP/Devices/Power/Battery.Android.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+namespace Windows.Devices.Power
+{
+	public partial class Battery
+	{
+		public static Battery AggregateBattery()
+		{
+			return new Battery();
+		}
```

Comment:
`AggregateBattery` is a property, not a method. This is very likely why CI fails.

Basically here is what I think have happened:

- Sync generator is run on CI
- It tries to find `AggregateBattery` *property*, but it doesn't find. So it generates one.
- We end up having two `AggregateBattery` members (method and property)
- Hence, compile error.

Reply:
Thanks.